To: vim_dev@googlegroups.com
Subject: Patch 7.3.1
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.119
Problem:    Build problem on Mac. (Nicholas Stallard)
Solution:   Use "extern" instead of "EXTERN" for p_vfile.
Files:	    src/option.h


*** ../vim-7.3.118/src/option.h	2011-02-09 15:59:32.000000000 +0100
--- src/option.h	2011-02-12 13:56:50.000000000 +0100
***************
*** 857,863 ****
  #ifdef IN_OPTION_C
  char_u	*p_vfile = (char_u *)""; /* used before options are initialized */
  #else
! EXTERN char_u	*p_vfile;	/* 'verbosefile' */
  #endif
  EXTERN int	p_warn;		/* 'warn' */
  #ifdef FEAT_CMDL_COMPL
--- 857,863 ----
  #ifdef IN_OPTION_C
  char_u	*p_vfile = (char_u *)""; /* used before options are initialized */
  #else
! extern char_u	*p_vfile;	/* 'verbosefile' */
  #endif
  EXTERN int	p_warn;		/* 'warn' */
  #ifdef FEAT_CMDL_COMPL
*** ../vim-7.3.118/src/version.c	2011-02-09 18:47:36.000000000 +0100
--- src/version.c	2011-02-12 13:59:41.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     119,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
237. You tattoo your email address on your forehead.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
